client/src/home/add_button/AddButton.js

```diff
@@ -29,13 +29,15 @@ function AddButton(props) {
 function AddItemForm(props) {
   // Input fields
   const [itemName, setItemName] = useState("");
-  const [itemId, setItemId] = useState("");
+  const [batchId, setBatchId] = useState("");
   const [lbPerHouse, setLbPerHouse] = useState(0);
   const [weight, setWeight] = useState(0);
   const [outByDate, setOutByDate] = useState(new Date());
 
   const [cookies] = useCookies(["token"]);
   const [showWeightWarning, setWeightWarning] = useState(false);
+  const [showitemNameWarning, setitemNameWarning] = useState(false);
+  const [showbatchIdWarning, setbatchIdWarning] = useState(false);
```

Comment:
Kind of a nitpick: maybe want to camel case `showitemNameWarning` and `showbatchIdWarning`